src/commands/remoteDebugJava/DebugProxy.ts

```diff
@@ -37,8 +37,8 @@ export class DebugProxy extends EventEmitter {
 
             this._server.on('connection', (socket: Socket) => {
                 if (this._wsclient) {
-                    ext.outputChannel.appendLog(`[Proxy Server] The server is already connected to "${this._wsclient.url.hostname}". Rejected connection to "${socket.remoteAddress}:${socket.remotePort}"`);
-                    this.emit('error', new Error(`[Proxy Server]  The server is already connected to "${this._wsclient.url.hostname}". Rejected connection to "${socket.remoteAddress}:${socket.remotePort}"`));
+                    ext.outputChannel.appendLog(`[Proxy Server] The server is already connected. Rejected connection to "${socket.remoteAddress}:${socket.remotePort}"`);
```

Comment:
Why change the error?

Reply:
When it updated the "@types/websocket" package, I guess "url" didn't exist anymore on "this._wsclient" so the build was failing. This isn't the most important piece of code, so I figured easiest way to fix the build was just remove that part of the message